lib/ansiblelint/utils.py

```diff
@@ -21,16 +21,36 @@
 import os
 import glob
 import imp
-import ansible.utils
-from ansible.playbook.task import Task
 import ansible.constants as C
 from ansible.module_utils.splitter import split_args
 import yaml
 from yaml.composer import Composer
 from yaml.constructor import Constructor
 
+try:
+    from ansible.utils import parse_yaml_from_file, path_dwim
+except ImportError:
+    from ansible.parsing import DataLoader
```

Comment:
`from ansible.parsing.dataloader import DataLoader` works from ansible commit [dcdcd9e9c518eafd533842664fe3c1545793a524](https://github.com/ansible/ansible/commit/dcdcd9e9c518eafd533842664fe3c1545793a524). If someone has odd version of ansible installed (from git).